### PR TITLE
Adds deployment test 

### PR DESCRIPTION
**Pull request by kapilkalra04**
## Description
 - Modifies install.sh to now utilise systemd service for correct tomcat installation inside the vagrant machine
 - Adds a deployment test for packet_data.pcap that is intended to be run inside the vagrant machine

## Motivation
 - Adds a rigorous deployment-test to check for any future code-breakage
 
 ## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checks
 - [x] Squash your commits down to one, or a few, discreet changesets before submitting a pull request
 - [x] Read the [contributing guide](https://github.com/prasadtalasila/BITS-Darshini/blob/master/.github/CONTRIBUTING.md)
 - [x] PR should be motivated, i.e. what does it fix, why, and if relevant how
 - [ ] If possible, include an example in the description, that could help all readers (including project members) to get a better picture of the change
 - [ ] Avoid other runtime dependencies unless absolutely necessary
 - [x] Rebase your commit history so that each commit is meaningful and states the intention behind the change
 - [x] Ensure that your pull request follows the [style guidelines](https://github.com/prasadtalasila/BITS-Darshini/wiki/Coding-Style) and [coding standards](https://github.com/prasadtalasila/BITS-Darshini/wiki/Coding-Standards)
 - [ ] At least one commit should mention `Fixes #<issue number>` _if relevant_

## Notify reviewers
@prasadtalasila

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--
+
+Have you read AutolabJS' Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/prasadtalasila/BITS-Darshini/blob/master/.github/CODE_OF_CONDUCT.md
```
Comment (prasadtalasila) [inline .github/ISSUE_TEMPLATE.md]: BITS Darshini's code of conduct

File: data/packet/test_files/locked_test_file.pcap [hunk 2]
```diff
@@ -0,0 +1,3 @@
+asdad
```
Comment (prasadtalasila) [inline data/packet/test_files/locked_test_file.pcap]: This file is not needed

File: scripts/install.sh [hunk 3]
```diff
@@ -64,14 +64,13 @@ sudo cp conf/tomcat.service /etc/systemd/system/
...
-#make tomcat a part of start-up scripts
-sudo update-rc.d tomcat defaults
-sudo update-rc.d tomcat enable
-
 #reload the systemd daemon so that it knows about our service file
 sudo systemctl daemon-reload
 sudo systemctl start tomcat
 
+#make tomcat a part of start-up scripts
+sudo systemctl enable tomcat.service
```
Comment (prasadtalasila) [inline scripts/install.sh:72]: shouldn't this come before the command on line-68?

**Review by prasadtalasila — CHANGES_REQUESTED** [review]
@kapilkalra04 please see the suggested changes.

File: scripts/install.sh (hunk 3 above)
Comment (kapilkalra04) [reply scripts/install.sh:72]: I think it should come after so as to let the system first know about the presence of a new file inside /etc/systemd/system

**Review by prasadtalasila — APPROVED** [review]

**Pull request merged**